scripts/genunicodetables.lua

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/lua
```

Comment:
Also regarding this line: IIRC FreeBSD installs lua to `/usr/local/bin`, so this should be `#!/usr/bin/env lua`.